src/services/bulkstatus.service.js

```diff
@@ -91,7 +91,7 @@ async function checkBulkStatus(req, res) {
               display: 'New resource created'
             }
           ],
-          text: 'Bulk import successfully completed'
+          text: `Bulk import successfully completed, successfully imported ${bulkStatus.successCount} resources`
```

Comment:
The wording of the test seems to imply that we won't always have a success count. In which case, this sentence will print a bit weird.

Reply:
We should always have a count unless the operation fails completely (then we shouldn't end up here) This count isn't based off the info provided by the export server, it's a count of the imports we were able to do